README.rst

```diff
@@ -1386,6 +1386,9 @@ An example of ``output_fn`` for the accept type "application/python-pickle" can
             # if the content type is not supported.
             pass  
 
+A example with the ``input_fn`` and ``output_fn`` above can be find in
```

Comment:
two small changes:
* remove 'the'
* s/find in/found

Reply:
ok